### PR TITLE
update shadow values. now with ∞ more umbra

### PR DESCRIPTION
**Pull request by morethanreal**
@shyndman @cdata @notwaldorf @nevir PTAL

### CODE REVIEW

File: shadow.html [hunk 1]
```diff
@@ -12,34 +12,79 @@
...
+
+    --shadow-elevation-2dp: {
+      box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.14),
+                  0 1px 5px 0 rgba(0, 0, 0, 0.12),
+                  0 3px 1px -2px rgba(0, 0, 0, 0.2);
+    };
+
     --shadow-elevation-1: {
-      box-shadow: 0 1px 4px 0 rgba(0, 0, 0, 0.37);
+      box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.14),
```
Comment (shyndman) [inline shadow.html]: Can you do this?
```
mixin(--shadow-elevation-2dp);
```

**Comment by shyndman** [comment]
Needs more umbra.

lgtm

File: shadow.html (hunk 1 above)
Comment (morethanreal) [inline shadow.html]: Cannot right now due to Polymer/polymer#1399 :(

**Comment by morethanreal** [comment]
FYI, I will also open PR's to elements using shadows to switch to the `-*dp` mixins.

File: shadow.html [hunk 2]
```diff
@@ -12,34 +12,49 @@
...
     };
 
-    --shadow-elevation-1: {
-      box-shadow: 0 1px 4px 0 rgba(0, 0, 0, 0.37);
+    /* from http://codepen.io/shyndman/pen/c5394ddf2e8b2a5c9185904b57421cdb */
+
+    --shadow-elevation-2dp: {
+      box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.14),
+                  0 1px 5px 0 rgba(0, 0, 0, 0.12),
+                  0 3px 1px -2px rgba(0, 0, 0, 0.2);
```
Comment (nevir) [inline shadow.html]: Lol, the `dp` is such a lie!

**Comment by nevir** [comment]
LGTM

**Pull request merged**